tests/test_api.rs

```diff
@@ -3400,7 +3400,8 @@ fn heap_statistics() {
   assert!(s.total_global_handles_size() >= s.used_global_handles_size());
   assert!(s.used_heap_size() > 0);
   assert!(s.heap_size_limit() >= s.used_heap_size());
-  assert!(s.peak_malloced_memory() >= s.malloced_memory());
+  assert!(s.peak_malloced_memory() > 0);
+  assert!(s.malloced_memory() > 0);
```

Comment:
@bnoordhuis this assert failed :shrug:
https://github.com/denoland/rusty_v8/runs/1033709642